frontend/src/scenes/session-recordings/components/RecordingRow.tsx

```diff
@@ -37,6 +40,22 @@ function ActivityScoreLabel({ score }: { score: number | undefined }): JSX.Eleme
         description = 'low'
     }
 
+    return { description, backgroundColor }
+}
+
+export function ActivityScoreLabel({
+    score,
+    clean = false,
+}: {
+    score: number | undefined
+    clean?: boolean
+}): JSX.Element {
+    const { description, backgroundColor } = getActivityScoreDescription({ score })
+
+    if (clean) {
+        return <>{description}</>
```

Comment:
should we have a tooltip here

"we use x and y and z to calculate a synthetic activity score"